### PR TITLE
Fix/autocomplete filter not behaving correctly using or operator page 4078

### PR DESCRIPTION
**Pull request by ingmar-stipriaan**

### CODE REVIEW

File: src/components/autocompleteInput.js [hunk 1]
```diff
@@ -112,19 +112,20 @@
     const belowMinimumMessage = useText(validationBelowMinimum);
     const helperTextResolved = useText(helperTextRaw);
     const modelProperty = getProperty(property || '') || {};
-    const labelProperty = getProperty(labelPropertyId) || {};
     const { modelId: propertyModelId, referenceModelId } = modelProperty;
     const { contextModelId } = model;
     const modelId =
       contextModelId || referenceModelId || propertyModelId || model || '';
+    const idProperty = getIdProperty(modelId) || {};
```
Comment (stefan-betty) [inline src/components/autocompleteInput.js:119]: how can `idProperty` be an empty `{}`?

Comment (ingmar-stipriaan) [reply src/components/autocompleteInput.js:119]: Well, if you don't have a modelId (when you drag it on your canvas), we need to have a fallback.

**Review by M-Ryan92 — APPROVED** [review]

File: src/components/autocompleteInput.js [hunk 2]
```diff
@@ -112,19 +112,20 @@
...
+    const labelProperty = getProperty(labelPropertyId) || idProperty;
```
Comment (JorisPannekeet) [inline src/components/autocompleteInput.js:120]: Why is a label either a label or an id?

File: src/components/autocompleteInput.js [hunk 3]
```diff
@@ -330,62 +328,67 @@
      */
     /* eslint-disable no-underscore-dangle */
 
-    const numberDebouncedInputValue = parseFloat(debouncedInputValue, 10);
-
-    // There is an empty string as fallback, because empty label is not a valid value to search on
-    const textDebouncedInputValue =
```
Comment (JorisPannekeet) [inline src/components/autocompleteInput.js:336]: I see you have removed the textDebouncedInputValue that was just recently added by Jordy, is that correct?

File: src/components/autocompleteInput.js (hunk 1 above)
Comment (ingmar-stipriaan) [reply src/components/autocompleteInput.js:119]: It is the property yes, not the id of the id property

**Review by stefan-betty — APPROVED** [review]

File: src/components/autocompleteInput.js [hunk 4]
```diff
@@ -330,62 +328,67 @@
      */
     /* eslint-disable no-underscore-dangle */
 
-    const numberDebouncedInputValue = parseFloat(debouncedInputValue, 10);
+    // After searching or setting a default value
+    if (debouncedInputValue) {
+      // ["relational_property_name", "property_name"]
```
Comment (ewoutquax) [inline src/components/autocompleteInput.js:333]: This looks like commented code; Can it be removed?

**Pull request merged**